scripts/install.js

```diff
@@ -0,0 +1,13 @@
+var sauceConnectLauncher = require('sauce-connect-launcher');
+var SAUCE_CONNECT_LAUNCHER_VERSION = '4.5.4';
+
+sauceConnectLauncher.download({
+  logger: console.log.bind(console),
+  version: SAUCE_CONNECT_LAUNCHER_VERSION,
+}, function (err) {
+  if (err) {
+    console.error(`Failed to download sauce connect binary: ${err}`);
+    console.log('sauce-connect-launcher will attempt to re-download next time it is run.');
```

Comment:
is this true? 
```console.log('sauce-connect-launcher will attempt to re-download next time it is run.');```

Reply:
Technically, it should be next it is installed (`npm install`) thanks for pointing out, I will make it strict.

Reply:
changed wording